spec/factories/factories.rb

```diff
@@ -0,0 +1,37 @@
+FactoryBot.define do
+  factory :customer do
+    first_name { Faker::Name.first_name }
+    last_name { Faker::Name.last_name }
+  end
+
+  factory :invoice_item do
+    item
+    invoice
+    quantity { Faker::Number.number(digits: 3) }
+    unit_price { Faker::Number.within(range: 100..20000) }
```

Comment:
if it going to produce a number with only 3 digits, should the number range be from 100 to 999?